bindings/java/src/main/native/jucx_common_def.cc

```diff
@@ -5,19 +5,23 @@
 
 #include "jucx_common_def.h"
 extern "C" {
+  #include <ucs/debug/assert.h>
   #include <ucs/debug/debug.h>
 }
 
 #include <string.h>    /* memset */
 #include <arpa/inet.h> /* inet_addr */
 
 
+static JavaVM *jvm_global;
+
 extern "C" JNIEXPORT jint JNICALL JNI_OnLoad(JavaVM *jvm, void* reserved) {
    ucs_debug_disable_signals();
+   jvm_global = jvm;
    return JNI_VERSION_1_1;
 }
 
-static inline void log_error(const char *error)
+inline void log_error(const char *error)
```

Comment:
just noticed this - no need to pass __FILE__ to ucs_error, also there is redundant space after %s
also, what is the point of making it inline if it is not declared in header file ?
I think it should be a macro and not an inline function